### PR TITLE
Fixing Issue #5184

### PR DESCRIPTION
**Pull request by davidair**

### CODE REVIEW

**Comment by dnkoutso** [comment]
@davidair sorry where is issue #5184 can you link to it?

**Comment by davidair** [comment]
https://github.com/CocoaPods/CocoaPods/issues/5184

File: spec/specification/set_spec.rb [hunk 1]
```diff
@@ -106,4 +106,20 @@ module Pod
...
+  #---------------------------------------------------------------------------#
+
+  describe 'Handling empty directories' do
+    before do
+      repos = [Source.new(fixture('spec-repos/test_empty_dir_repo'))]
+      @set = Source::Aggregate.new(repos).search_by_name('EmptyDir_spec').first
+    end
+
+    it 'raises when encountering empty directories' do
+      @set.name.should == "EmptyDir_spec"
```
Comment (dnkoutso) [inline spec/specification/set_spec.rb]: nit: single quotes

File: spec/specification/set_spec.rb [hunk 2]
```diff
@@ -106,4 +106,20 @@ module Pod
...
+      exception = lambda { @set.specification }.should.raise Informative
+      exception.message.should.include "Could not find the highest version for `EmptyDir_spec`."
```
Comment (dnkoutso) [inline spec/specification/set_spec.rb]: nit: same  single quotes

**Comment by dnkoutso** [comment]
Seems like a few tests are failing

**Comment by davidair** [comment]
Yes, I think they are picking up the empty repo I've created. Taking a look...

**Comment by davidair** [comment]
Tests seem to be passing now, except  Ruby: 2.0.0-p451 and Ruby: 2.0.0-p247. Is this expected?

**Comment by dnkoutso** [comment]
@davidair ah yes this needs to be fixed here. PR coming up

**Comment by dnkoutso** [comment]
https://github.com/CocoaPods/Core/pull/403

**Comment by dnkoutso** [comment]
@davidair rebase should be good now.

**Comment by davidair** [comment]
Trying to squash the changes properly, pardon my bad git

**Pull request merged**